src/win32_platform.h

```diff
@@ -49,13 +49,13 @@
 #endif
 
 // GLFW requires Windows XP or later
-#if WINVER < 0x0501
+#if WINVER < 0x0604
  #undef WINVER
- #define WINVER 0x0501
+ #define WINVER 0x0604
 #endif
-#if _WIN32_WINNT < 0x0501
+#if _WIN32_WINNT < 0x0604
  #undef _WIN32_WINNT
- #define _WIN32_WINNT 0x0501
+ #define _WIN32_WINNT 0x0604
```

Comment:
These need to stay at their current values. Look below for how to handle missing parts of the platform SDK on older environments.